src/lib/markbind/src/parser.js

```diff
@@ -341,7 +325,11 @@ Parser.prototype._preprocess = function (node, context, config) {
         }
         actualContent = self._rebaseReferenceForStaticIncludes(actualContent, element, config);
       }
-      element.children = cheerio.parseHTML(actualContent, true); // the needed content;
+      const inlineWrapper = isInline ? 'span' : 'div';
```

Comment:
The constant name is not consistent with the behaviour since `isInline` can be false, and the resulting wrapper is meant for non-inline includes.

Shall we rename `inlineWrapper` to `wrapper` or `data-included-from-wrapper` ?

Reply:
Sometimes I wonder how confused I must have been what I wrote the code... I don't even know why it's named inlineWrapper haha
I'll change it to something like `wrapperType`. Thanks so much.